stock_available_immediately/product.py

```diff
@@ -20,42 +20,24 @@
 ##############################################################################
 
 from openerp.addons import decimal_precision as dp
-
 from openerp.osv import orm, fields
 
 
-class product_immediately_usable(orm.Model):
+class ProductTemplate(orm.Model):
     """
-    Inherit Product in order to add an "immediately usable quantity"
-    stock field
     Immediately usable quantity is : real stock - outgoing qty
     """
-    _inherit = 'product.product'
+    _inherit = 'product.template'
 
```

Comment:
Not strictly necessary, following the tendency of odoo to put most product.product fields in the template.